src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -47,9 +47,10 @@
       %li.nav-item.scrollable-tab-link
         = link_to('Build Results', '#build-results', class: 'nav-link text-nowrap', 'aria-controls': 'build-results',
                   'aria-selected': 'false', 'data-toggle': 'tab', role: 'tab')
-      %li.nav-item.scrollable-tab-link#changes-item{ 'data-request-number': @bs_request.number, 'data-request-action-id': @actions.first[:id] }
-        = link_to('Changes', '#changes', class: 'nav-link text-nowrap', 'aria-controls': 'changes', 'aria-selected': 'false',
-                  'data-toggle': 'tab', role: 'tab')
+      - unless @action[:sourcediff].nil?
```

Comment:
This sounds a bit  too expensive to just to ask the action if it has a diff or not. We know which actions that can possibly have a diff and which do not.

Reply:
I've removed it